misc-utils/lsclocks.c

```diff
@@ -205,6 +212,38 @@ static clockid_t parse_clock(const char *name)
 	errx(EXIT_FAILURE, _("Unknown clock: %s"), name);
 }
 
+static int64_t get_namespace_offset(const char *name)
+{
+	char *tokstr, *buf, *saveptr, *line, *space;
+	uint64_t ret;
+	int fd;
+
+	fd = open(_PATH_PROC_TIMENS_OFF, O_RDONLY);
+	if (fd == -1)
+		err(EXIT_FAILURE, _("Could not open " _PATH_PROC_TIMENS_OFF));
+
+	read_all_alloc(fd, &buf);
+
+	for (tokstr = buf; ; tokstr = NULL) {
+		line = strtok_r(tokstr, "\n", &saveptr);
+		if (!line)
+			errx(EXIT_FAILURE, _("Did not find timezone offset"));
```

Comment:
Maybe we can use `continue` here (rather than an error message) as it's already used later if `name` is not found.